scripts/update_config.py

```diff
@@ -8,11 +8,17 @@
 
 # Mapping - key: old name, value: new name
 name_dict = {'task_patience':'lr_patience',
-               'do_train': 'do_pretrain',
-               'train_for_eval':'do_target_task_training',
-               'do_eval': 'do_full_eval',
-               'train_tasks':'pretrain_tasks',
-               'eval_tasks':'target_tasks'}
+				'do_train': 'do_pretrain',
+				'train_for_eval':'do_target_task_training',
+				'do_eval': 'do_full_eval',
+				'train_tasks':'pretrain_tasks',
+				'eval_tasks':'target_tasks', 
+				'eval_data_fraction': 'target_train_data_fraction', 
+				'eval_val_interval': 'target_train_val_interval',
+				'eval_max_vals': 'target_train_max_vals',
+				'load_eval_checkpoint': 'load_target_train_checkpoint', 
+				'eval_data_fraction': 'target_train_data_fraction'
+				}
```

Comment:
Odd indentation...

Reply:
Git seems to be displaying the indentation incorrectly here....